src/Dropdown.php

```diff
@@ -2990,7 +2990,11 @@ public static function getDropdownValue($post, $json = true)
                         // Ensure condition contains table name to prevent ambiguity with fields from `glpi_entities` table
                         $where["$table.$key"] = $value;
                     } else {
-                        $where[$key] = $value;
+                        if ($key === 'NOT' && isset($where['NOT'])) {
```

Comment:
I guess the fix is incomplete. Issue will be the same with key different than `NOT`. Also, I'm not entirely sure we should reach this case here.